data/mods/ssb/moves.js

```diff
@@ -163,6 +163,36 @@ let BattleMovedex = {
 		type: "Electric",
 		zMovePower: 200,
 	},
+	// Aethernum
+	cataclysm: {
+		accuracy: 90,
+		basePower: 140,
+		category: "Physical",
+		desc: "Resets all of the user's boosts to 0, then Atk, Def and Speed get lowered by 1",
+		shortDesc: "Clears user's boosts; lowers Atk, Def and Spe.",
+		id: "cataclysm",
+		name: "Cataclysm",
+		isNonstandard: "Custom",
+		pp: 5,
+		priority: 0,
+		flags: {contact: 1, protect: 1, mirror: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, "Earth Power", target);
+			this.add('-anim', source, "Continental Crush", target);
+			this.add('-anim', source, "Giga Impact", target);
```

Comment:
How long does this animation chain take to run? If its too long id ask to cut a move.